features-json/web-app-manifest.json

```
@@ -1,6 +1,6 @@
 {
   "title":"Add to home screen (A2HS)",
-  "description":"The ability for a user to \"install\" a website and use it as if it was a natively installed app. To enable this behaviour, a website must serve a valid Web App Manifest.",
+  "description":"The ability for a user to \"install\" a website and use it as if it was a natively installed app. Metadata for the web app like name, links to icons, preferred URL, etc can be configured in the Web App Manifest.",
```

Just to clarify a little bit, the ability to "install" is just one capability provided by Web Manifest: web manifest can be used to, for instance, set the theme-color of a web page without installing the application first. The manifest can also be used in other contexts (e.g., it could hypothetically be used to populate a new tab page). 

```suggestion
  "description":"The ability for a user agent make use of web manifest metadata, which includes things like the theme-color, name, links to icons, start URL, etc. for a web application.",
```